internal/internal.go

```diff
@@ -14,6 +14,16 @@
 
 package internal
 
+import "time"
+
 // UserAgent is the user agent to be added to the outgoing
 // requests from the exporters.
 const UserAgent = "opencensus-go-v0.1.0"
+
+// The monotonic clock is used in subtractions hence
+// the duration since start added back to start gives
+// end as a monotonic time.
+// See https://golang.org/pkg/time/#hdr-Monotonic_Clocks
+func MonoticTimeSince(start time.Time) time.Time {
```

Comment:
name here seems a little strange. "since" seems to imply that it returns a duration. What about "MonotonicEndTime"?

Reply:
In deed, thanks for catching that. However, the new name deosn't convey how we calculate the EndTime as being from time.Now, how about an update to your suggestion `MonotonicTimeNow`?

Reply:
Actually, never mind, I'll go with your suggestion, I withdraw my previous reservations.